presto-main/src/main/java/io/prestosql/server/protocol/ExecutingStatementResource.java

```diff
@@ -301,7 +301,7 @@ public Response cancelQuery(
     }
 
     @DELETE
-    @Path("/v1/statement/partialCancel/{queryId}/{stage}/{slug}/{token}")
+    @Path("/v1/statement/executing/partialCancel/{queryId}/{stage}/{slug}/{token}")
```

Comment:
this could potentially break automation for users?

Reply:
I shouldn't.  The spec for these URIs says you get them from the statement resource document by following the next uris.